packages/cli/src/cli.ts

```diff
@@ -47,22 +47,35 @@ export function cli({
                         description:
                             'List of tables to be included in the generation',
                     })
-                    .array('tables');
+                    .array('tables')
+                    .config('configFile')
+                    .parserConfiguration({
+                        'dot-notation': false,
+                        'strip-aliased': true,
+                    });
             },
             async (argv) => {
                 console.warn('Connecting to database schema...');
 
-                const { schema } = await generateSchema({
+                const result = await generateSchema({
                     connectionString: argv.connectionString,
                     out: argv.out,
                     defaultSchema: argv.defaultSchema,
                     schemas: argv.schemas,
                     tables: argv.tables,
+                    schemaDefOverrides: argv.schemaDefOverrides,
                 });
-                const tables = Object.keys(schema.properties ?? {});
-                console.warn(`Generated schema with ${tables.length} table(s)`);
-                for (const table of tables) {
-                    console.warn(`- ${table}`);
+
+                if (result) {
+                    const tables = Object.keys(result.schema.properties ?? {});
+
+                    console.warn(
+                        `Generated schema with ${tables.length} table(s)`,
```

Comment:
You should provide some basic feedback to the user in case the overrides are passed but not actually applied to anything.